Source/WebCore/html/HTMLElement.cpp

```diff
@@ -1124,6 +1126,25 @@ bool HTMLElement::shouldExtendSelectionToTargetNode(const Node& targetNode, cons
     return true;
 }
 
+ExceptionOr<Ref<ElementInternals>> HTMLElement::attachInternals()
+{
+    auto* queue = reactionQueue();
+    if (!queue)
+        return Exception { NotSupportedError };
```

Comment:
Can we give it a useful error message? e.g. "attachInternals is only supported on a custom element instance"